src/CloudSecrets.UnitTests/ConnectedModeSecretsTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿/*
```

Comment:
Please let me know if this & the other class were supposed to go into the folder CloudSecrets or not.